editor/index.html

```diff
@@ -42,6 +42,7 @@
 		<script src="js/libs/ternjs/doc_comment.js"></script>
 		<script src="js/libs/tern-threejs/threejs.js"></script>
 		<script src="js/libs/signals.min.js"></script>
+		<script src="../examples/js/vr/HelioWebXRPolyfill.js"></script>
```

Comment:
How much longer are we going to need this polyfill for?

Reply:
The next release of Helio (0.98.10) will remove the need for the polyfill.

Reply:
Not sure on exact timing